python/py-chainer/Portfile

```diff
@@ -38,5 +40,17 @@ if {${name} ne ${subport}} {
         }
     }
 
+    variant hdf5 description {Support HDF5 serialization} {
+        depends_lib-append      port:py${python.version}-h5py
+    }
+
+    variant imagedataset description {Support image dataset} {
+        depends_lib-append      port:py${python.version}-Pillow
+    }
+
+    variant protobuf description {Caffe model support} {
+        depends_lib-append      port:py${python.version}-protobuf3
+    }
+
     livecheck.type      none
```

Comment:
Not your change, but I'm not sure why `livecheck.type` is set to `none`?

Reply:
that's okay, typically there is no livecheck for the py??-subports, only for ```py-<port>```